ppb/engine.py

```diff
@@ -73,12 +89,20 @@ def run(self):
 
     @property
     def current_scene(self):
+        """
+        Get current scene
+        """
         try:
             return self.scenes[-1]
         except IndexError:
             return None
 
     def activate(self, next_scene: dict):
+        """
+        Activates a specific scene
```

Comment:
Should be written as a command. (Drop the "s" on "activates"). Specific is probably unneeded.